src/native-common/ScrollView.tsx

```diff
@@ -82,7 +83,15 @@ export class ScrollView extends ViewBase<RX.Types.ScrollViewProps, RX.Types.Stat
             scrollHandler = undefined;
         }
 
-        const keyboardShouldPersistTaps = (overrideKeyboardShouldPersistTaps || this.props.keyboardShouldPersistTaps ? 'always' : 'never');
+        // keyboardShouldPersistTaps can be overridden, a deprecated boolean that should translate to 'always' or 'never', or 'handled'
+        let keyboardShouldPersistTaps: ScrollViewProps['keyboardShouldPersistTaps'] = 'never'; // never is the default
+        if (overrideKeyboardShouldPersistTaps ||
+            ((typeof this.props.keyboardShouldPersistTaps === 'boolean') && (this.props.keyboardShouldPersistTaps))) {
+            keyboardShouldPersistTaps = 'always';
+        }
+        if (typeof this.props.keyboardShouldPersistTaps === 'string') {
```

Comment:
I think this should be an `else if` since overrideKeyboardShouldPersistTaps should probably still override even if a string value is provided . (not 100% confident in this though)

Reply:
I am fairly confident I handle the various cases, but I do not like my structure there either, and any reader should feel 100% confident, so I'll take another cut at it, taking into account that I can also simplify per your note above. Thanks for the review